cluster-sync/sync.sh

```diff
@@ -96,8 +106,15 @@ function wait_cdi_available {
 }
 
 function configure_uploadproxy_override {
-  host_port=$(./cluster-up/cli.sh ports uploadproxy | xargs)
-  override="https://127.0.0.1:$host_port"
+  if [[ $KUBEVIRT_PROVIDER =~ kind.* ]]; then
+    # To enable port mapping, it must be configured both in the Kind configuration and the uploadProxyURLOverride.
+    # We use the environment variable KIND_PORT_MAPPING to ensure the setup is applied in both locations.
+    IFS=':' read -r container_port host_port <<< "$KIND_PORT_MAPPING"
```

Comment:
Oh, this IFS thing is really a recipe for trouble in my experience.. but if you insist don't forget to do something like
https://www.unix.com/302179290-post2.html

Reply:
I will use other way to achieve this